cocos/2d/CCLabel.h

```diff
@@ -364,6 +364,11 @@ class CC_DLL Label : public Node, public LabelProtocol, public BlendProtocol
     void enableBold();
 
     /**
+     * set bold effect, use freetype to support bold only for ttf
+     */
+    void setBold(bool bold);
```

Comment:
I suggest not providing another `setBold` API, we should replace the old Bold implementation for TTF.